software/teams/teams.go

```diff
@@ -46,3 +52,75 @@ func Get(teamID string, usersEnabled bool, client houston.ClientInterface, out i
 
 	return nil
 }
+
+// retrieves all teams
+func List(client houston.ClientInterface, out io.Writer) error {
+	var teams []houston.Team
+	var cursor string
+	count := -1
+
+	for len(teams) < count || count == -1 {
+		resp, err := client.ListTeams(cursor, ListTeamLimit)
+		if err != nil {
+			return err
+		}
+		count = resp.Count
+		teams = append(teams, resp.Teams...)
+		cursor = teams[len(teams)-1].ID
+	}
+
+	teamsTable := printutil.Table{
+		Padding:        []int{50, 50},
+		DynamicPadding: true,
+		Header:         []string{"TEAM ID", "TEAM NAME"},
+		ColorRowCode:   [2]string{"\033[1;32m", "\033[0m"},
+	}
+	for i := range teams {
+		teamsTable.AddRow([]string{teams[i].ID, teams[i].Name}, false)
+	}
+	return teamsTable.Print(out)
+}
+
+func Update(teamID, role string, client houston.ClientInterface, out io.Writer) error {
+	if role == "" {
```

Comment:
see my previous comment in the cmd package, we should probably do the cmd validation (require role or default to `None`) in the cmd package, and here we can always expect a role to be passed.